src/player.js

```diff
@@ -24,6 +25,7 @@ export default class Player {
     this.createInputButtonPair = createInputButtonPair;
     this.createShoppingCartItem = createShoppingCartItem;
     this.extractBandFollowInfo = extractBandFollowInfo;
+    this.extractFanTralbumData = extractFanTralbumData;
```

Comment:
what is tralbum? It looks like this is a thing and not a typo, so curious what it means

Reply:
it's Bandcamp's internal term for a track or album (`tr`rack`album`)